tasks/test-tutorial

```diff
@@ -18,20 +18,25 @@ if [ -z "$1" ]
   then
   echo "You have not supplied a path to a Redwood project."
   echo "We will create one for you."
-  echo "We will copy './packages/create-redwood-app/template' and run 'yarn rw upgrade --tag canary'"
+  echo "We will copy './packages/create-redwood-app/template' and the packages/*"
 
   TMP_DIR=$(mktemp -d -t redwood)
-  cd ../../packages/create-redwood-app
-  yarn babel-node dist/create-redwood-app.js $TMP_DIR
 
-  cd $TMP_DIR
-  yarn rw upgrade --tag canary
+  cd ../../packages/create-redwood-app
+  yarn babel-node dist/create-redwood-app.js $TMP_DIR  --no-yarn-install
 else
   echo "You have supplied a path $1, we will not create a new "
   echo "Redwood project, we will use the app you have specified."
   TMP_DIR=$1
 fi
 
+cd "$SCRIPTPATH/../"
+# build all the packages
+yarn build
```

Comment:
What do you think of doing a clean here first? yarn build:clean && yarn build

Might cover some weird situations with switching branches

Reply:
Totes!

Reply:
I'm actually going to script the TS builds since they're so slow. We don't need them for testing the tutorial.